hawtio-web/src/main/webapp/app/jvm/js/connect.ts

```diff
@@ -159,6 +159,21 @@ module JVM {
       angular.extend($scope.currentConfig, $scope.connectionConfigs[$scope.lastConnection]);
       Core.$apply($scope);
     };
+    
+	var autoconnect = $location.search();
+	if (typeof autoconnect != 'undefined' && typeof autoconnect.name != 'undefined') {	
+		var conOpts = Core.createConnectOptions({
+		        scheme: 'http',
```

Comment:
I wonder if it would be possible to change this feature to allow to set the scheme as well, since now auto connect link limits to only http connections. It would be good to have scheme parameter in url as well.